css-animations-2/Overview.bs

```diff
@@ -847,6 +873,271 @@ expressed in milliseconds, it must be divided by 1,000 to produce a value in
 seconds before being assigned to the {{AnimationEvent/elapsedTime}} member of
 the {{AnimationEvent}}.
 
+# Animation Triggers # {#triggers}
+
+An <dfn export>animation trigger</dfn> is used to control the playback
+of its associated [=animation=] for time-driven animations.
+[=Animation triggers=] follow the same [[web-animations-1#hierarchical|hierarchy]]
+as animations and are associated with a [=timeline=] and an [=animation trigger effect|effect=].
```

Comment:
This makes it sound like animation triggers are separate things from animations, but I suspect we should say that as a part of [updating the current time of animations](https://drafts.csswg.org/web-animations-1/#update-animations-and-send-events) we evaluate whether a trigger condition has been reached and apply a stateful change to the animation.

Reply:
Sounds good. I'll work on that